File: 19aug19/functions/eight.c
#include<stdio.h>
void funca();
extern void funcb();
int ga = 10;
extern int gb;
void main()
	{
	extern int ma;
	ma = ga + gb + ma;
	printf("ma=%d\n", ma);
	funcb();
	return;
	}
void funca()
	{	
	}

File: digital_twin/day2/functions/eight.py
def greet():
    print("namste")
    print("good day")

def funca(x, y):
    return 2*x + y

def funcb(a=10, b=20, *c):
    print("a =", a)
    print("b =", b)
    print('c =', c)

def funcb(a=10, b=20, *args):
    print("a =", a)
    print("b =", b)
    print('c =', c)
